targets/frontend/src/components/layout/Nav.js

```diff
@@ -104,6 +104,11 @@ export function Nav() {
               <NavLink>Thèmes</NavLink>
             </Link>
           </Li>
+          <Li>
+            <Link href="/kali/blocs" passHref>
```

Comment:
can you use <ActiveLink> (work like next's Link)

Reply:
?